### PR TITLE
Add ability to override the testing web content root using environmen…

### PR DESCRIPTION
**Pull request by remcomulder**
Amended pull request with changes included from dev team feedback - https://github.com/aspnet/Mvc/pull/8004

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactory.cs [hunk 1]
```diff
@@ -161,6 +166,19 @@ private void SetContentRoot(IWebHostBuilder builder)
             }
         }
 
+        private bool TrySetContentRootUsingEnvironmentVariables(IWebHostBuilder builder)
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactory.cs]: UsingEnvironmentVariables? FromConfig?

**Comment by Tratcher** [comment]
FYI you don't have to open a new PR every time you make changes, you can push more commits to the same branch, or even force push.

**Comment by remcomulder** [comment]
Sorry to prod .. but can the other reviewers confirm if they're happy with these changes?  There's a growing number of people becoming frustrated that our products aren't working together.  We've been trying to get this issue resolved since early June.

**Comment by pranavkm** [comment]
Thanks for the PR @remcomulder! Merged your changes in https://github.com/aspnet/Mvc/commit/28c0c4d128fae199d73f2f62ce5736482e273891

**Pull request closed without merging**